client/src/main/java/io/pravega/client/segment/impl/AsyncSegmentInputStreamImpl.java

```diff
@@ -51,6 +51,7 @@
     private final AtomicBoolean closed = new AtomicBoolean(false);
     private final Controller controller;
     private final String delegationToken;
+    private final long requestId;
```

Comment:
This is a strange sort of 'requestId' it's being used more like 'writerId' on the write side in that it is not identifying the request, but rather the reader in this case.

Reply:
Yes, I will rename it to `requestorId`. 
This is used to identify the commands originating from a segment writer / segment reader and this is also used to ensure the responses (which also have this id)from the SSS are routed back to the correct reply processors of these segment writers/ segment readers.